db/migrate/20151214213221_update_braintree_customer.rb

```diff
@@ -0,0 +1,10 @@
+class UpdateBraintreeCustomer < ActiveRecord::Migration
+  def change
+    rename_column :payment_braintree_customers, :card_vault_token, :default_payment_method_token
+    change_table :payment_braintree_customers do |t|
+      t.remove :card_unqiue_number_identifier,
```

Comment:
@Tuuleh I'd save removing these fields for another PR. A lot of code depends on `Payment.customer` which searches by `email` - this change is going to break a lot of things. What if you first changed how `Payment.customer` works?